config/default.json

```diff
@@ -40,20 +40,20 @@
       "0xd03ea8624c8c5987235048901fb614fdca89b117"
     ]
   },
-  "useReviewerWhitelist": true,
-  "useDelegateWhitelist": true,
-  "useProjectOwnerWhitelist": true,
+  "useReviewerWhitelist": false,
+  "useDelegateWhitelist": false,
+  "useProjectOwnerWhitelist": false,
   "authentication": {
-    "secret": "fc04928dc74759168bf7cbc58bd2e43983e050342a82d25af4b93396d904020006b84e67cb45030fe906db9039d4aba720b6dc68564a3577cb1e6e34c9c8a1b6cbc6e3f824ead946ab9f66180647deb5ad097378b1e40440dc9e102a8113b904931481133a80aac801a2f65970e02edaf579b4cfee10cf704d86adbb819cae14578cc4d9f126bae6cad65ae4ad243c550f4af42ad743b2d4061ec54b80e64978a98a0283d99ee2489b9059ca9ac0681f4bd54eb8052ab2a11a5b96e58a171dde225e182913ae4cb60c34cc8b54f93986ffa2c57284a3cd2710170e0cfd272f303d6852447d6a2863fb644d02465b2f2be667c9fbd7798b4f5cc2e70749840f2c",
-    "strategies": [
+    "entity": "user",
+    "service": "users",
+    "secret": "fc04928dc74759168bf7cbc58bd2e43983e050342a82d25af4b93396d904020006b84e67cb45030fe906db9039d4aba720b6dc68564a3577cb1e6e34c9c8a1b6cbc6e3f824ead946ab9f66180647deb5ad097378b1e40440dc9e102a8113b904931481133a80aac801a2f65970e02edaf579b4cfee10cf704d86adbb819cae14578cc4d9f126bae6cad65ae4ad243c550f4af42ad743b2d4061ec54b80e64978a98a0283d99ee2489b9059ca9ac0681f4bd54eb8052ab2a11a5b96e58a171dde225e182913ae4cb60c34cc8b54f93986ffa2c57284a3cd2710170e0cfd272f303d6852447d6a2863fb644d02465b2f2be667c9fbd7798b4f5cc2e70749840f2c=",
+    "authStrategies": [
       "jwt",
       "web3"
     ],
-    "path": "/authentication",
-    "service": "users",
-    "jwt": {
+    "jwtOptions": {
       "header": {
-        "type": "access"
+        "typ": "access"
```

Comment:
Is it correct?

Reply:
Yes, it's correct. For more information follow the link below.
https://docs.feathersjs.com/api/authentication/service.html#configuration

Reply:
Note: typ in the header options is not a typo, it is part of the JWT JOSE header specification